pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerDebug.java

```diff
@@ -272,7 +272,10 @@ public String getServerRoutingStats() {
   }
 
   private long getRequestId() {
-    return _requestIdGenerator.getAndIncrement();
+    if (REQUEST_ID_GENERATOR.get() == Long.MAX_VALUE) {
```

Comment:
nit: Do we really need to handle this? Even at 1M QPS, it would take 10^18 / 10^6 = 10^12 seconds for a broker to hit this.

Also, technically there's a race condition here.

Reply:
True, removing the check